cmd/csi-driver/Dockerfile

```diff
@@ -1,13 +1,13 @@
 FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5
 
 RUN microdnf update -y && rm -rf /var/cache/yum
-ADD CentOS-Base.repo /etc/yum.repos.d/
+ADD AlmaLinux-Base.repo /etc/yum.repos.d/
 RUN microdnf install -y systemd && microdnf install -y cryptsetup
 
 LABEL name="HPE CSI Driver for Kubernetes" \
       maintainer="HPE Storage" \
       vendor="HPE" \
-      version="2.0.0" \
+      version="2.1.1.0" \
```

Comment:
Is this a new convention with four positions?

Reply:
Yes, agreed internally with the team unless it brings up any issues. We will know more once we work on the beta helm chart

Reply:
If the 4th position is going to be package version, is the container considered the package in this instance then? I think in this case the "version" will be the visible "application version" in the partner ecosystems.

Reply:
So should we go with application version 2.1.1 and package version 2.1.1.0 ?